src/image_modifications.cpp

```diff
@@ -951,16 +948,16 @@ REGISTER_MOD_PARSER(CROP, args)
 
 	SDL_Rect slice_rect { 0, 0, 0, 0 };
 
-	slice_rect.x = lexical_cast_default<int16_t, const std::string&>(slice_params[0]);
+	slice_rect.x = lexical_cast_default<int16_t, std::string_view>(slice_params[0]);
 
 	if(s > 1) {
-		slice_rect.y = lexical_cast_default<int16_t, const std::string&>(slice_params[1]);
+		slice_rect.y = lexical_cast_default<int16_t, std::string_view>(slice_params[1]);
```

Comment:
Does `lexical_cast` actually work for a `string_view`…? It feels like it shouldn't…

Reply:
Planning on replacing these with utils::from_chars anyway.

Reply:
I _was_ wondering why you didn't do that as well.